README.md

```diff
@@ -135,6 +147,67 @@ a = 0-5;
 write |a|; // 5
 ```
 
+## Comparisons
+
+DanskDiego has `true` and `false` as boolean constants, and `null` for usage with records or arrays.
+
+Equality operators in Diego are similar to most other imperative languages, being `==` for equality, `!=` for inequality.
+Additionally, the language has `<`, `>`, `<=`, `>=` for equality checks.
+
+### Control flow
+
+#### If-statements
+
+If-statements in DanskDiego can be followed a else-statement, or have the else omitted. The predicate must be enclosed by
+a parenthesis. Predicates can be chained using the `&&` or the `||` operators.
+
+```
+if (i == 0) write 1;
```

Comment:
if also use a `then` delimiter to separate the condition from the statement body. See the factorial example in "tests"